types/searchjs/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for searchjs 1.0
+// Project: https://github.com/deitch/searchjs
+// Definitions by: Melvin Mupondori <https://github.com/iammerus>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function setDefaults(options: any): void;
+
+export function resetDefaults(): void;
+
+export function singleMatch(field: any, s: any, text: any, word: any, regexp: any, start: any, end: any): any;
```

Comment:
Is there any possibility that you can fill these in with something more specific than `any`?

Reply:
I filled in all that have concrete types with more specific types